src/views/AddItem.jsx

```diff
@@ -1,3 +1,68 @@
+// LIBRARY IMPORTS
+import React, { useState } from 'react';
+
+//LOCAL IMPORTS
+import { addItem } from '../api/firebase.js';
+
 export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+	// SET STATES
+	const [itemName, setItemName] = useState('');
+	const [days, setDays] = useState(0);
+	const [status, setStatus] = useState(null);
+
+	// HANDLE EVENTS
+	const handleItemNameChange = (e) => {
+		setItemName(e.target.value);
+	};
+	const handleDaysChange = (e) => {
+		setDays(e.target.value);
+		console.log(`day updated: ${days}`);
+	};
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+
+		let itemData = {
+			itemName: itemName,
+			daysUntilNextPurchase: days,
+		};
+
+		try {
+			await addItem('my test list', itemData);
+			setStatus('This item has been added to your list!');
+			setItemName('');
+			setDays(0);
+		} catch (error) {
+			setStatus("Oh no, this item wasn't added");
+		}
+	};
+
+	return (
+		<div>
+			<form onSubmit={handleSubmit}>
+				<label htmlFor="itemName">
+					Item name:
+					<input
+						type="text"
+						id="itemName"
+						onChange={handleItemNameChange}
+						value={itemName}
+					/>
+				</label>
+				<br />
+				<label>
+					How soon will you buy this again?
+					<select onChange={handleDaysChange} value={days}>
+						<option value="">Please choose an option</option>
```

Comment:
Right now there's an issue where the user can submit without changing this option and the database says that the date of next purchase is the same as the date the item was created.

Reply:
This is an issue with using a `select` instead of a radio dial but wanting to present an "unselected" value. 

The easiest solution is to remove the first option entirely. You could also add form validation to ensure that value is not nothing before submitting ( we will add additional form validation in a later issue ).

Other CSS-based options are described here https://stackoverflow.com/questions/8605516/set-the-select-option-as-blank-as-default-in-html-select-element